locked-asset/simple-lock/src/proxy_farm.rs

```diff
@@ -220,25 +239,26 @@ pub trait ProxyFarmModule:
         let lp_proxy_token_payment = EsdtTokenPayment::new(
             lp_proxy_token.get_token_id(),
             farm_proxy_token_attributes.farming_token_locked_nonce,
-            exit_farm_result.initial_farming_tokens.amount,
-        );
-        self.send().direct_esdt(
-            &caller,
-            &lp_proxy_token_payment.token_identifier,
-            lp_proxy_token_payment.token_nonce,
-            &lp_proxy_token_payment.amount,
+            initial_farming_tokens.amount,
         );
+        self.send()
+            .direct_non_zero_esdt_payment(&caller, &lp_proxy_token_payment);
+
+        let reward_tokens = exit_farm_result.get_reward_tokens();
+        self.send()
+            .direct_non_zero_esdt_payment(&caller, &reward_tokens);
```

Comment:
It would be preferable to have one multiTransfer call.

Reply:
Done.